go.mod

```diff
@@ -40,6 +40,7 @@ require (
 	github.com/shirou/gopsutil/v3 v3.20.11
 	github.com/syncthing/notify v0.0.0-20201210100135-17de26665ddc
 	github.com/syndtr/goleveldb v1.0.1-0.20200815071216-d9e9293bd0f7
+	github.com/thejerf/suture v4.0.0+incompatible
 	github.com/thejerf/suture/v4 v4.0.0
```

Comment:
We should probably have just one suture dependency?

Reply:
idk why but my IDE and go command line just go mad about this stuff. Lemme back out all the go.mod changes.